pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -270,13 +275,17 @@ func createDnsmasqConfiglet(
 		}
 	}
 	if netconf.DhcpRange.Start != nil {
-		dhcpRange = netconf.DhcpRange.Start.String()
-	}
-	if isIPv6 {
-		file.WriteString(fmt.Sprintf("dhcp-range=::,static,0,60m\n"))
+		dhcpStart := netconf.DhcpRange.Start.String()
+		dhcpEnd := netconf.DhcpRange.End.String()
+		file.WriteString(fmt.Sprintf("dhcp-range=%s,%s,%s,60m\n",
+			dhcpStart, dhcpEnd, ipv4Netmask))
```

Comment:
Does this change any dnsmasq behavior the way we use it?
You no longer include the "static" part which is what we set since all assignments we want to do have an entry in the dhcp-hostsdir.

Reply:
This was a temporary hack, the "static" definition is in place now.